R/learn.R

```diff
@@ -82,7 +82,8 @@ dm_learn_from_db <- function(dest, dbname = NA, schema = NULL, name_format = "{t
     ungroup() %>%
     pull_tbl()
 
-  tables <- map2(table_info$from, table_info$vars, ~ tbl(con, dbplyr::ident_q(.x), vars = .y))
+  table_info$from_id <- DBI::dbUnquoteIdentifier(con, DBI::SQL(table_info$from))
```

Comment:
We need to use `dbUnquoteIdentifier()` because `group_by()` doesn't work with S4 classes.